content/en/docs/release-oversight/improving-ci-signal.md

```diff
@@ -0,0 +1,125 @@
+
+Our corpus/history (n) will be well over 250, so use that line.  Our sample size is 10.  Find the corpus pass percentage of the job,test tuple.  Then go straight up and find the Y value.  That value is the pass percent required to be 95% sure that the current payload is worse than the corpus (history).
+
+
+### FAQ
+
+#### Who is watching for payload regressions and opening reverts?
+For now, the [Technical Release Team](/docs/release-oversight/the-technical-release-team) watches payload results on the current release branch.  They will open the revert PRs and engage the involved teams (any team that contributed a PR to the payload that regressed).
+
+#### How will we be contacted if our PR is part of a regressed payload?
+TRT will ping the relevant team slack aliases in #forum-release-oversight.  
+
+#### Who approves the revert for merge?
+In order to ensure they are included in the conversation/resolution, we would like the team that delivered the original PR to /approve the reversion.  However, if they are unavailable or unresponsive this may be escalated to the staff-engineering team to ensure we do not remain in a regressed state longer than necessary.
+
+#### What happens if a giant change like a rebase with high business value and on the critical path of the release regresses the pass rate? Will it be reverted? Will we work on it until it passes good enough? What if the kubelet build hours or days later causes the regression?
+Since we have a way to run the payload promotion checks before a PR merges, we encourage high risk changes to run the payload acceptance before they merge.  If that payload acceptance test fails, then it is 95% likely (math, not gut) that the PR is reducing our reliability.  That signal is enough to engage other teams if necessary, but since most repos other than openshift/kubernetes are owned by a single team and that team is the local expert, in the majority of cases the expert should be local.
+
+If the PR isn’t pre-checked and we catch it during payload promotion, then even a high business value PR is subject to reversion.  The unrevert PR is the place to talk about expected and observed impact to reliability versus the new feature.  Both the revert and the unrevert are human processes, so there is room for discussion.
+
+#### What does an “identified fix” look like?
+To avoid merging the revert, teams will need to be able to point to an open PR that contains a fix, and which has passed the payload acceptance checks.
+
+The payload acceptance check aggregates the result of N runs into a pass/fail percentage and compares it against the baseline.  We are going to start by only failing overall when there is a 95% confidence that the PR is worse than the baseline.  Here is a chart showing the calculation.  The exact pass/fail counts vary depending on how bad the baseline is.
```

Comment:
nit: Drop "Here is a chart showing the calculation." and/or link up to the Fischer's subsection?

Reply:
Thanks!